apicast/conf.d/apicast.conf

```diff
@@ -86,6 +86,7 @@ location = /_threescale/oauth_store_token {
   internal;
   proxy_set_header  X-Real-IP  $remote_addr;
   proxy_set_header  X-Forwarded-For $proxy_add_x_forwarded_for;
+  proxy_set_header  Host  "$backend_host";
```

Comment:
I haven't tried without, but I think it wasn't working.
All other calls to the backend do already have this header.